### ISSUE TITLE
yamon v4.0.7 stops collecting data after a couple days

### ISSUE THREAD

**Issue by zsarosi** [al-caughey/YAMon-v4#27]
Hi
It is not the same issue as #21 !
I noticed that my Yamon v4.0.7 stops collecting data after a couple of days. The webpage is still working, it just notifies, that the hourly data is missing and the monthly data has gaps in it. After restarting yamon from the routers command line it works again for some days.
I have Yamon running on my OpenWRT v19.07.1 r10911-c155900f66 Linksys WRT1900ACS router.
The USB Stick is mounted under /opt, it is working, it has no issues.

Edit on Nov. 20. 2020:
I followed the problem, and I decided not to restart Yamon each time it stops collecting data. I noticed that after some time - one two or more days - it continues collecting data. I see (one, two or more days long) holes in the diagrams. 
Strange, am I the only one having this problem?

**Comment by m0jek** [comment]
No, you are not. I haven't had time to check this out properly, but I have the same problem. I thought it has something to do with the corn jobs that are supposed to refresh / roll teh monitoring. Need to do some more checking.
I can add that YAMon 4.07 was working perfectly for months and slowly began to go funny.

**Comment by zsarosi** [comment]
Is there a workaround, i.e. monitoring a file or process and restart yamon if it is necessary?
